web/cobrands/fixmystreet-uk-councils/assets.js

```diff
@@ -197,6 +199,57 @@ fixmystreet.assets.brent.not_found = function(layer) {
     fixmystreet.message_controller.road_found(layer);
 };
 
+fixmystreet.assets.brent.no_asset_message_defaults = {
+    'Parks_and_Open_Spaces' : '#js-brent-park-services',
+    'Highways' : '#js-brent-road-services'
+};
+
+fixmystreet.assets.brent.parkRoadIsFound = {
+    park: false,
+    road: false,
+    isParkorRoad: function() { return this.park || this.road; }
+};
+
+fixmystreet.assets.brent.found_filter = function(layer) {
+    if (fixmystreet.reporting.selectedCategory().category === 'Grass verges / shrub beds - littering') {
+        if (layer.features[0] && layer.features[0].type === 'Parks_and_Open_Spaces') {
+            fixmystreet.assets.brent.parkRoadIsFound.park = true;
+        }
+        if (layer.features[0] && layer.features[0].type === 'Highways') {
+            fixmystreet.assets.brent.parkRoadIsFound.road = true;
+        }
+        if (fixmystreet.assets.brent.parkRoadIsFound.isParkorRoad()) {
+            fixmystreet.message_controller.road_found(layer);
+        }
+    } else {
+        fixmystreet.message_controller.road_found(layer);
+    }
+};
+
+fixmystreet.assets.brent.not_found_filter = function(layer) {
+    if (fixmystreet.reporting.selectedCategory().category === 'Grass verges / shrub beds - littering') {
+        layer.fixmystreet.no_asset_msg_id = '#js-brent-parks-and-highways';
+        if (layer.features[0] && layer.features[0].type === 'Parks_and_Open_Spaces') {
```

Comment:
Rather than having to check if there are any features, all the type checks could be replaced by `layer.fixmystreet.http_options.params.TYPENAME` which is always present.
(Or you could give the layer a name in config and check layer.name :) )